projects/js-packages/publicize-components/src/components/media-section/index.js

```diff
@@ -118,92 +136,96 @@ export default function MediaSection() {
 		[ updateAttachedMedia ]
 	);
 
-	const renderPreview = useCallback( () => {
-		if ( isVideo( metaData.mime ) ) {
-			// TBD
-			return <div>Video Preview</div>;
-		}
-
-		const { width, height, sourceUrl } = mediaData;
-
-		if ( width && height && sourceUrl ) {
-			return (
-				<ResponsiveWrapper naturalWidth={ width } naturalHeight={ height } isInline>
-					<img src={ sourceUrl } alt="" />
-				</ResponsiveWrapper>
-			);
-		}
-	}, [ mediaData, metaData ] );
+	const renderPreview = useCallback(
+		open => {
+			if ( isVideo( metaData.mime ) ) {
+				// TBD
+				return <div>Video Preview</div>;
+			}
+
+			const { width, height, sourceUrl } = mediaData;
+
+			if ( width && height && sourceUrl ) {
+				return (
+					<div className={ styles[ 'preview-wrapper' ] }>
+						<button className={ styles.remove } onClick={ onRemoveMedia }>
+							<VisuallyHidden>{ __( 'Remove media', 'jetpack' ) }</VisuallyHidden>
```

Comment:
Is accessibility the purpose of the `VisuallyHidden` component?

Reply:
Yup!